src/components/equipment/Equipments.tsx

```diff
@@ -39,7 +39,7 @@ const columns: Column<PartialEquipment>[] = [
   {
     title: 'Instruments',
     render: (rowData) =>
-      rowData.equipmentInstruments.length
+      rowData.equipmentInstruments?.length
```

Comment:
probably no need to check for `length` property as `rowData.equipmentInstruments` would be sufficient

Reply:
It could be an empty array and I think it is better to check for length to be sure that we show `-` when there are no instruments.